var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -53,6 +53,7 @@ class PyPillow(PythonPackage):
     depends_on('jpeg')
     depends_on('py-pytest', type='test')
     depends_on('py-pytest-runner', type='test')
+    depends_on('imagemagick', type='test')
```

Comment:
I don't have imagemagick installed on my system, but the tests still pass for me. Is this dependency documented anywhere?

Reply:
I do have imagemagick installed on the system but not in spack. See Tests/helper.py in Pillow.

Reply:
Ah, I bet the issue is that since I don't have imagemagick, it skips those tests, but since you do have it, they are run and fail. Must be an issue with older versions of imagemagick. In that case, yes, we should add the dependency.